.github/workflows/openapi-diff.yml

```diff
@@ -0,0 +1,47 @@
+name: openapi-diff
+on:
+  push:
+    branches:
+      - master
```

Comment:
@billyvg Do you know if this is like on.push.branches AND on.push.paths, or OR? 

Also in either case, does this action really need to run on push @NisanthanNanthakumar?

Reply:
@joshuarli The consensus, for now, is that we should run on push to master to check that nothing unexpected happened on master, similar to `acceptance.yml`

Reply:
Okay.

Reply:
@joshuarli actually not sure, I assumed it was AND but I don't know off hand